l1-contracts/test/merkle/Merkle.t.sol

```diff
@@ -27,4 +51,125 @@ contract MerkleTest is Test {
       assertEq(merkle.computeRoot(), frontier.root(), "Frontier Roots should be equal");
     }
   }
+
+  function testNaiveSiblingPathAndMembershipVerification(uint256 _idx) public view {
```

Comment:
The focus here is on testing that the membership verification works and not on the naive tree so would just call this testVerifyMembership.